packages/cli/src/commands/config/set.ts

```diff
@@ -28,12 +24,15 @@ export default class Set extends BaseCommand {
     const res = await this.parse(Set)
     const curr = readConfig(this.config.configDir)
     const node = res.flags.node ?? curr.node
-    const gasCurrency = res.flags.gasCurrency
-      ? (gasOptions as any)[res.flags.gasCurrency as string]
-      : curr.gasCurrency
-    writeConfig(this.config.configDir, {
-      node,
-      gasCurrency,
-    })
+    const gasCurrency = res.flags.gasCurrency as StrongAddress
+
+    await writeConfig(
+      this.config.configDir,
+      {
+        node,
+        gasCurrency,
+      },
+      await this.getKit()
```

Comment:
hold up. it used to default to he current config if no gasCurrency was passed in. but will it now?

Reply:
nope this was lost in the refactoring, fixing it